include/irr/builtin/glsl/bxdf/common.glsl

```diff
@@ -135,7 +135,7 @@ irr_glsl_IsotropicViewSurfaceInteraction irr_glsl_calcFragmentShaderSurfaceInter
 {
     irr_glsl_IsotropicViewSurfaceInteraction interaction;
     interaction.V.dir = _View;
-    interaction.V.dPosdScreen[0] = dFdx(_SurfacePos);
+    interaction.V.dPosdScreen[0] = dFdx(_SurfacePos);// TODO: require the user to feed us the derivatives
```

Comment:
@Crisspl this is a problem when including the file in other shaders than fragment shader